Nexus.Sdk.Token/Responses/TokenOperationResponse.cs

```diff
@@ -77,4 +77,13 @@ public TokenOperationResponse(string code, string hash, OperationAccountResponse
         [JsonPropertyName("blockchainTransactionId")]
         public string BlockchainTransactionId { get; set; }
     }
+
+    public class UpdateOperationStatusRequest
```

Comment:
Why is this in the Response class?

Reply:
Please move it to Requests

Reply:
[Refactor code to move request code to requests file](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/55/commits/36675aff91474e96bc1c66e9dddb3605baa42928)